js/sectionCollector.js

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable no-use-before-define */
+// Book class : book presentation
+// eslint-disable-next-line max-classes-per-file
```

Comment:
Consider not disabling the linters, instead resolve the errors you are having to ensure that you are following best practices.